packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -1041,6 +1042,127 @@ curl -v -X POST \
 }
 ```
 
+#### Add Smart Card Identity Provider
+
+
+Adds a new Smart Card `X509` type IdP to your organization
+
+##### Request Example
+
+> **Notes:** You must first add the IdP's server certificate to the IdP key store before you can add a Smart Card `X509` IdP with a `kid` credential reference. You need to upload the whole trust chain as single key using the [Key Store API](#add-x-509-certificate-public-key).
+Depending on the information stored in the smart card select proper template "idpuser.subjectAltNameEmail" or "idpuser.subjectAltNameUpn".
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "X509",
+  "status": "ACTIVE",
+  "features": [],
+  "name": "Smart Card IDP Name",
+  "protocol": {
+    "type": "MTLS",
+    "credentials": {
+      "trust": {
+        "revocation": "CRL",
+        "revocationCacheLifetime": 2880,
```

Comment:
In the UI the max lifetime is 72 hrs, here we are showing 48 hrs. We should add in the document that maximum lifetime supported is 72 hrs

Reply:
It is documented in the MTLS trust section.